julia.html.markdown

```diff
@@ -190,7 +190,7 @@ end
 # inside the julia folder to find these files.
 
 # You can initialize arrays from ranges
-a = [1:5] # => 5-element Int64 Array: [1,2,3,4,5]
+a = [1:5;] # => 5-element Int64 Array: [1,2,3,4,5]
```

Comment:
In julia 0.4:

    julia> [1:5]
    WARNING: [a] concatenation is deprecated; use [a;] instead